src/api/api.js

```diff
@@ -0,0 +1,62 @@
+export const API_SERVER = 'https://kdt-frontend.programmers.co.kr/documents';
+
+export const request = async (url = '', options = {}) => {
+  try {
+    const res = await fetch(`${API_SERVER}${url}`, {
+      ...options,
+      headers: {
+        'x-username': 'juyeon',
+        'Content-Type': 'application/json',
+      },
+    });
+
+    if (res.ok) {
+      return await res.json();
+    }
+
+    throw new Error('API 처리중 오류 발생');
+  } catch (error) {
+    console.log(error);
+  }
+};
+
+export const getAllDocuments = async () => {
+  const documents = await request('');
```

Comment:
```suggestion
  const documents = await request();
```
해당 `request`의 인자가 비어있으면 빈문자열이 들어가도록 되어있는데 여기서 굳이 한번더 빈문자열을 넣어줄 필요성은 없을것으로 보여집니다.